src/components/AddBook.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+
+import { addBook } from '../redux/books/books';
+
+const AddBook = () => {
+  const dispatch = useDispatch();
+
+  const [title, setBookTitle] = useState('');
+  const [author, setBookAuthor] = useState('');
+
+  const handleBookTitle = (e) => {
+    setBookTitle(e.target.value);
+  };
+
+  const handleBookAuthor = (e) => {
+    setBookAuthor(e.target.value);
+  };
+
+  const addNewBook = () => {
+    if (title !== '' && author !== '') {
+      dispatch(addBook)({ id: uuidv4(), title, author });
+      setBookTitle('');
+      setBookAuthor('');
+    }
```

Comment:
As asked by the previous reviewer we can not add or remove a book, please correct them.